### PR TITLE
About component

### LINKED ISSUE #26: About Component
image
- import title component
- logo SVG is under public folder
- include clouds
- /components/home/About.jsx

### PR DESCRIPTION
**Pull request by zalam007**

### CODE REVIEW

**Comment by seanquiambao** [comment]
add an image for your component and link your issue under development

File: src/components/home/About.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const About = () => {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex items-center px-32 z-1">
+        <Image src={navlogo} alt="Picture of navlogo" />
+
+        <div className="flex-col">
+          <Title text="About us" />
+          <div className="text-center w-1/2 mx-auto mt-10">
+            Indian Student Association-ISA at UCR serves the purpose to provide
```
Comment (seanquiambao) [inline src/components/home/About.jsx:21]: make the text size a little bigger

File: src/components/home/About.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            a commonplace for people to share and exchange Indian culture of all
+            regions in India. We strive to provide a place for all Indians and
+            non-Indians to come to socialize, celebrate festivals, enjoy food,
+            meet new people, and learn more about Indian culture
+          </div>
+        </div>
+      </div>
+
+      <Image src={cloud} alt="Picture of cloud" className="mt-[-58px] z-0" />
```
Comment (seanquiambao) [inline src/components/home/About.jsx]: image

the cloud doesn't fill the page width-wise

File: src/components/home/About.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+const About = () => {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex items-center px-32 z-1">
+        <Image src={navlogo} alt="Picture of navlogo" />
+
+        <div className="flex-col">
+          <Title text="About us" />
+          <div className="text-center w-1/2 mx-auto mt-10">
```
Comment (seanquiambao) [inline src/components/home/About.jsx]: you can put the whole CSS to its parent tag in line 14

File: src/app/page.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
 import Landing from "@/components/home/Landing";
+import About from "@/components/home/About";
 
 const Home = () => {
   return (
     <>
+      <About />
```
Comment (seanquiambao) [inline src/app/page.js]: place this below the landing page

File: src/components/home/About.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+import Image from "next/image";
+import cloud from "@/public/assets/cloud.svg";
+import navlogo from "@/public/images/navlogo.webp";
+
+const About = () => {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex items-center px-32 z-1">
+        <Image src={navlogo} alt="Picture of navlogo" />
```
Comment (seanquiambao) [inline src/components/home/About.jsx]: scale logo a little bigger, you can use tailwindCSS to scale the image

File: src/components/home/About.jsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+import Title from "@/components/Title";
+
+import Image from "next/image";
+import cloud from "@/public/assets/cloud.svg";
+import navlogo from "@/public/images/navlogo.webp";
+
+const About = () => {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex items-center px-32 z-1">
```
Comment (seanquiambao) [inline src/components/home/About.jsx]: image
logo should be closer to the texts

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by zalam007** [comment]
update
ab

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**